src/plugins/orangehrmRecruitmentPlugin/Dao/CandidateDao.php

```diff
@@ -211,4 +212,24 @@ public function saveCandidateHistory(CandidateHistory $candidateHistory): Candid
         $this->persist($candidateHistory);
         return $candidateHistory;
     }
+
+    /**
+     * @param int $candidateId
+     * @return int
+     */
+    public function getInterviewCountByCandidateId(int $candidateId): int
+    {
+        $candidateInterviews = $this->getRepository(Interview::class)->findBy(['candidate'=> $candidateId]);
+        return count($candidateInterviews);
```

Comment:
better to use query builder. don't load entities to get count

Reply:
fixed